cogs/other/mobs.py

```diff
@@ -300,7 +300,8 @@ async def spawn_event(self, ctx):
 
             await ctx.send(embed=embed)
 
-            await self.db.update_user(u.id, "health", u_health)
+            if u_health <= u_db['health']:
```

Comment:
why less than or equal to, why unnecessarily update the database if the value in the database is the same

Reply:
because when you fight multiple mobs you can regen health, for instance you fight a spider and you end the fight with 1hp and at the same time end a fight with a creeper and you ahve 20hp youll end up with 20hp. thats why we need the less than or equal to